conjureup/controllers/newcloud/common.py

```diff
@@ -178,14 +178,15 @@ def setup_lxdbr0_network():
                     n, out.stderr.decode()))
 
 
-async def do_bootstrap(creds, msg_cb, fail_msg_cb, region=None):
+async def do_bootstrap(creds, msg_cb, fail_msg_cb):
     if not app.is_jaas:
         app.log.info('Bootstrapping Juju controller.')
         msg_cb('Bootstrapping Juju controller.')
         track_event("Juju Bootstrap", "Started", "")
         cloud_with_region = app.current_cloud
-        if region:
-            cloud_with_region = '/'.join([app.current_cloud, region])
+        if app.current_region:
+            cloud_with_region = '/'.join([app.current_cloud,
+                                          app.current_region])
```

Comment:
This code also exists in `app.py`, should we move that code to the `tui.py` for this controller?